server/config/index.js

```diff
@@ -32,6 +34,19 @@ function anyEnabled() {
 	} );
 }
 
+const ssrConfig = template(
+	'var data = <%= data %>;' +
+	'<%= config %>' +
+	'<%= isEnabled %>' +
+	'<%= anyEnabled %>'
+) ( {
+	data: JSON.stringify( clientData ),
+	config: config.toString(),
```

Comment:
I think I managed to refactor it to use shared config API. See https://github.com/Automattic/wp-calypso/pull/11352/commits/0fd7456166b81e56915f61396b4570b4b79a5983.